shuttle/templates/schedules/load_schedule.html

```diff
@@ -1,29 +1,32 @@
 {% block body_content %}
 
-<div class="small-card">
-    <div class="card-body log-body scrolling-wrapper">
-        <table class="table table-hover">
-            <tbody>
-            {% for i in range(schedule|length) %}
-                <tr>
-                    {% for j in range(schedule[i]|length) %}
-                        {% if j % 2 == 0 %}
-                            <td class="even-column">
-                                {% else %}
-                            <td>
+    <div class="center-page">
+            <h3>Shuttle Schedule</h3>
+    </div>
+    <div class="small-card">
+        <div class="card-body log-body scrolling-wrapper">
+            <table class="table table-hover">
+                <tbody>
+                {% for i in range(schedule|length) %}
+                    <tr>
+                        {% for j in range(schedule[i]|length) %}
+                            {% if j % 2 == 0 %}
+                                <td class="even-column">
+                                    {% else %}
+                                <td>
+                            {% endif %}
+                        {% if i == 0 %}
+                            <h7 class="schedule-time"><b>{{ schedule[i][j] }}</b></h7>
+                        {% else %}
+                            <h7 class="schedule-time">{{ schedule[i][j] }}</h7>
                         {% endif %}
-                    {% if i == 0 %}
-                        <h7 class="schedule-time"><b>{{ schedule[i][j] }}</b></h7>
-                    {% else %}
-                        <h7 class="schedule-time">{{ schedule[i][j] }}</h7>
-                    {% endif %}
-                    </td>
-                    {% endfor %}
-                </tr>
-            {% endfor %}
-            </tbody>
-        </table>
+                        </td>
+                        {% endfor %}
+                    </tr>
+                {% endfor %}
```

Comment:
![Screen Shot 2020-08-03 at 2 52 37 PM](https://user-images.githubusercontent.com/26125143/89221513-03e0d400-d599-11ea-9d05-841d4490757b.png)

Remove the extra column at the bottom of the table

Reply:
I'm not seeing an extra column on my screen... only the scrollbar at the bottom of the table

Reply:
Ahh I guess that is the scrollbar. Is there anyway to hide it when it is not necessary?

Reply:
I was wondering that too... I'll look into that